pyfar/_utils.py

```diff
@@ -0,0 +1,49 @@
+from pyfar.classes.warnings import PyfarDeprecationWarning
+import warnings
+
+
+# Decorator function to rename parameters to be deprecated
+def rename_arg(arg_map, warning_message):
+    """
+    Function for deprecating or renaming arguments.
```

Comment:
A minor, picky comment: Please add a new line after the first line of the Docstring :)